gulpfile.babel.js

```diff
@@ -120,75 +103,81 @@ gulp.task('copy:license', () =>
 gulp.task('copy:main.css', () => {
   const banner = `/*! HTML5 Boilerplate v${pkg.version} | ${pkg.license} License | ${pkg.homepage} */\n\n`;
 
-  gulp.src(`node_modules/main.css/dist/main.css`)
+  return gulp.src('node_modules/main.css/dist/main.css')
     .pipe(plugins().header(banner))
     .pipe(plugins().autoprefixer({
-      browsers: ['last 2 versions', 'ie >= 9', '> 1%'],
       cascade: false
     }))
     .pipe(gulp.dest(`${dirs.dist}/css`));
 });
 
 gulp.task('copy:misc', () =>
   gulp.src([
-
     // Copy all files
     `${dirs.src}/**/*`,
 
     // Exclude the following files
     // (other tasks will handle the copying of these files)
     `!${dirs.src}/css/main.css`,
     `!${dirs.src}/index.html`
-
   ], {
-
-    // Include hidden files by default
-    dot: true
-
-  }).pipe(gulp.dest(dirs.dist))
+      // Include hidden files by default
+      dot: true
+    }).pipe(gulp.dest(dirs.dist))
 );
 
 gulp.task('copy:normalize', () =>
   gulp.src('node_modules/normalize.css/normalize.css')
     .pipe(gulp.dest(`${dirs.dist}/css`))
 );
 
-gulp.task('modernizr', (done) =>{
-
+gulp.task('modernizr', (done) => {
   modernizr.build(modernizrConfig, (code) => {
     fs.writeFile(`${dirs.dist}/js/vendor/modernizr-${pkg.devDependencies.modernizr}.min.js`, code, done);
   });
-
 });
 
 gulp.task('lint:js', () =>
   gulp.src([
-    'gulpfile.js',
     `${dirs.src}/js/*.js`,
     `${dirs.test}/*.js`
   ]).pipe(plugins().jscs())
     .pipe(plugins().eslint())
     .pipe(plugins().eslint.failOnError())
 );
 
-
 // ---------------------------------------------------------------------
 // | Main tasks                                                        |
 // ---------------------------------------------------------------------
+gulp.task(
+  'copy',
+  gulp.series(
+    'copy:.htaccess',
+    'copy:index.html',
+    'copy:jquery',
+    'copy:license',
+    'copy:main.css',
+    'copy:misc',
+    'copy:normalize'
+  )
+);
+
+gulp.task(
+  'build',
+  gulp.series(
+    gulp.parallel('clean', 'lint:js'),
+    'copy',
+    'modernizr'
+  )
+);
 
-gulp.task('archive', (done) => {
-  runSequence(
+gulp.task(
+  'archive',
+  gulp.series(
     'build',
     'archive:create_archive_dir',
-    'archive:zip',
-    done);
-});
-
-gulp.task('build', (done) => {
-  runSequence(
-    ['clean', 'lint:js'],
-    'copy', 'modernizr',
-    done);
-});
+    'archive:zip'
+  )
+);
 
-gulp.task('default', ['build']);
+gulp.task('default', gulp.series('build'))
```

Comment:
eslint complains that this should have a semicolon at the end here.

Reply:
fixed